packages/components/CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 -   `Navigator`: add more pattern matching tests, refine existing tests ([47910](https://github.com/WordPress/gutenberg/pull/47910)).
 -   `ToolsPanel`: Refactor Storybook examples to TypeScript ([47944](https://github.com/WordPress/gutenberg/pull/47944)).
 -   `ToolsPanel`: Refactor unit tests to TypeScript ([48275](https://github.com/WordPress/gutenberg/pull/48275)).
+-   `Guide`: Convert to TypeScript ([#47493](https://github.com/WordPress/gutenberg/pull/47493)).
```

Comment:
Looks like this needs to be moved again under the newer `Unreleased` section of the CHANGELOG

Reply:
Yes. Fixed via https://github.com/WordPress/gutenberg/pull/47493/commits/737ab84b5473d7126c87f9f069f1193cc5233dea